docs/monitoring/08-05-send-notifications.md

```diff
@@ -44,8 +44,8 @@ Follow these steps to configure notifications for Slack every time Alertmanager
 
    | Parameter | Description |
    |-----------|--------------------|
-   | **global.alertTools.credentials.slack.channel** | Specifies the Slack channel which receives notifications on new alerts, such as `test-monitoring-alerts`.
-   | **global.alertTools.credentials.slack.apiurl** | Specifies the URL endpoint which sends alerts triggered by Prometheus rules. The Incoming WebHooks application provides you with the Webhook URL, such as `https://hooks.slack.com/services/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`, that you can paste in this configuration. |
+   | **global.alertTools.credentials.slack.channel** | Specifies the Slack channel, which receives notifications on new alerts, such as `test-monitoring-alerts`.
```

Comment:
```suggestion
   | **global.alertTools.credentials.slack.channel** | Specifies the Slack channel which receives notifications on new alerts, such as `test-monitoring-alerts`.
```
Also as above, I would say that the second part of the sentence is a defining one - this is this particular channel that receives the notifications on new alerts. https://dictionary.cambridge.org/grammar/british-grammar/relative-clauses-defining-and-non-defining. It can also be `that receives` :)